### PR TITLE
fix: bug of uploading avatar

### PR DESCRIPTION
**Pull request by aac3476**
caused by:
https://github.com/suyuan32/simple-admin-vben5-ui/commit/00844e8a4c5b0bc990c034e49fb7ee62115847a2

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

## Checklist

> ℹ️ Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](contributing.md).

- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs:dev` command.
- [ ] Run the tests with `pnpm test`.
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/simple-admin-core/src/components/form/ImageUpload.vue [hunk 1]
```diff
@@ -87,7 +87,7 @@ const props = defineProps({
   },
 });
 
-const emits = defineEmits(['update:value']);
+const emits = defineEmits(['change', 'update:value', 'delete']);
```
Comment (suyuan32) [inline apps/simple-admin-core/src/components/form/ImageUpload.vue:90]: 这两个事件没触发，vben5 用不太到这两个事件才删的

Comment (aac3476) [reply apps/simple-admin-core/src/components/form/ImageUpload.vue:90]: > 这两个事件没触发，vben5 用不太到这两个事件才删的

image

我这里不加的话有bug

Comment (suyuan32) [reply apps/simple-admin-core/src/components/form/ImageUpload.vue:90]: 没改动之前会这样嘛，我记得测试过没问题的，和这两个事件有啥关系

Comment (aac3476) [reply apps/simple-admin-core/src/components/form/ImageUpload.vue:90]: > 没改动之前会这样嘛，我记得测试过没问题的，和这两个事件有啥关系

不会，你线上的demo不会这样，但是那个应该不是最新版本，最新版本拉下来就这样，加了两个事件就正常了

Comment (suyuan32) [reply apps/simple-admin-core/src/components/form/ImageUpload.vue:90]: 好吧，好像是有问题，我再检查下

**Pull request merged**